### ISSUE TITLE
Explanations for the following MCQ

### ISSUE THREAD

**Issue by qweiping31415** [nus-cs2103-AY1920S1/forum#199]
Can I ask why the following questions have their respective answers as stated in the answer scheme? Tried checking the textbook for an explanation.

A4. Testing is a [verification | validation] type QA activity. (Ans:B)

A7. A coding standard [can contain rules that can be objectively enforced | may contain rules that are subjective]. (Ans: B)

A33. AddressBook-level4 uses continuous deployment. (Ans: B)

**Comment by damithc** [comment]
> A4. Testing is a [**verification** | validation] type QA activity. (Ans:B)

Consider different testing types (unit testing etc.). Some aim for verification while some aim for validation.

> A7. A coding standard [**can contain rules that can be objectively enforced** | may contain rules that are subjective]. (Ans: B)

Not everything in a coding standard can be objectively enforced. Some rules are subjective.

> A33. AddressBook-level4 uses continuous deployment. (Ans: B)

It does not release a new jar file every time new code is committed.

**Comment by nexolute** [comment]
> It does not release a new jar file every time new code is committed.

But could we argue that its true because Travis automatically deploy our gh pages?

**Comment by damithc** [comment]
> But could we argue that its true because Travis automatically deploy our gh pages?

Good point @nexolute. But that's just the documentation. Nevertheless, it can be argued that CD is being used in the project, although not for the product itself. I'll update the question to match.
